CRM/Contribute/Tokens.php

```diff
@@ -47,16 +40,17 @@ protected function getApiEntityName(): string {
   }
 
   /**
-   * Get a list of tokens for the entity for which access is permitted to.
+   * Get a list of tokens which are loaded via APIv4.
    *
    * This list is historical and we need to question whether we
    * should filter out any fields (other than those fields, like api_key
    * on the contact entity) with permissions defined.
    *
    * @return array
+   *   Ex: ['foo', 'bar_id', 'bar_id:name', 'bar_id:label']
    */
-  protected function getExposedFields(): array {
-    return [
+  protected function getApiTokens(): array {
```

Comment:
So once we resolve https://lab.civicrm.org/dev/core/-/issues/2745 my expectation is we would entirely remove this function - because there is nothing (yet) in this token class that should not be purely metadata driven